### PR TITLE
Add map sessions endpoint

### PR DESCRIPTION
**Pull request by elhardoum**

### CODE REVIEW

File: Endpoints/player/MapsEndpoint/MapsEndpoint.cs [hunk 1]
```diff
@@ -308,6 +307,47 @@ public Model.Maps GetSimple(OLabDBContext context, uint id)
...
+        throw new OLabUnauthorizedException(Utils.Constants.ScopeLevelMap, mapId);
+
+      Maps map = GetSimple(dbContext, mapId);
+
+      if (map == null)
+        throw new OLabObjectNotFoundException(Utils.Constants.ScopeLevelMap, mapId);
+
+      IList<SessionInfo> sessions = await dbContext.SessionInfo.FromSqlInterpolated(@$"
+        select s.uuid,
+               cast(from_unixtime(s.start_time) as datetime) as `Timestamp`,
```
Comment (RotateAt60MPH) [inline Endpoints/player/MapsEndpoint/MapsEndpoint.cs:332]: A hard-coded SQL state should not be necessary if EF relationships between entities are set up correctly.  Please investigate.

**Review by RotateAt60MPH — CHANGES_REQUESTED** [review]
A hard-coded SQL state should not be necessary if EF relationships between entities are set up correctly. Please investigate.

**Review by elhardoum — COMMENTED** [review]
Updated the query to use LINQ, though the relations are still not fully used (e.g. Session.SessionTraces vs a join)? I can send make another update to spend more time on this to use the relationships instead of joining. At least now it doesn't query the tables via hardcoded names/cols, but rather via the EF properties.

**Comment by RotateAt60MPH** [comment]
As discussed:

var userSession = await dbContext.UserSessions
  .AsNoTracking()
  .Include(x => x.UserSessionTraces)
  .FirstOrDefaultAsync(x => x.Id == mapId && x.UserId == user.Id && x.Issuer == user.Iss );

**Review by elhardoum — COMMENTED** [review]
This is now completed. When matching users, the auth issuer will be safely checked, so DT will now also see anonymous users in the session report. I was excluding them before not knowing they were external plays.

**Review by RotateAt60MPH — APPROVED** [review]

**Pull request merged**